index.html

```diff
@@ -15,7 +15,12 @@
     <header>
```

Comment:
- [ ] [REQUIRED] Good job so far! The width value of your homepage section seems to be increased because the layout of the home page elements looks flat. Also, the color of `My logo` is not the same as [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848), please give it the desired color value(Correct color: `#6070FF`) 😃 

- ### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/199759922-d6258f27-f0be-4e04-a69e-0479adada5e0.png)

- ### Figma design ✔️ 

![image](https://user-images.githubusercontent.com/63915024/199759889-56ec65ee-01dd-492d-b2b0-470b7c7bee72.png)

Reply:
Fixed.